### PR TITLE
Pre-release prep. for Chapel 1.13.0 : increment version numbers

### PR DESCRIPTION
**Pull request by awallace-cray**
work in progress

### CODE REVIEW

**Comment by awallace-cray** [comment]
for discussion w @ben-albrecht

**Comment by ben-albrecht** [comment]
Looks good. We'll also be updating `GOALS` and `CHANGES` with a 1.13 section, separately. It might be worth running the tests and/or doing some careful grepping to confirm that no tests depend on the version number being 1.12

**Comment by ben-albrecht** [comment]
Just for reference, #2524 is the previous version update PR. Note that `GOALS` and `CHANGES` were prematurely updated in that PR.

**Comment by awallace-cray** [comment]
@ronawho, whats the best way to test this?

**Comment by awallace-cray** [comment]
no comment

**Comment by ronawho** [comment]
No idea, maybe run the scripts that test the reatime tarball?

File: test/compflags/bradc/printstuff/version.goodstart [hunk 1]
```diff
@@ -1 +1 @@
- Version 1.12.0
\ No newline at end of file
+ Version 1.13.0
```
Comment (bradcray) [inline test/compflags/bradc/printstuff/version.goodstart]: I think the newly added linefeed at the end of this line may break this test (or perhaps another that depends on it?)  I.e., I think the lack of a linefeed in the previous version was intentional.

**Comment by bradcray** [comment]
Thanks for jumping on this, Tony.

When I've reviewed this in the past, I've typically just looked at the previous release (like the PR that Ben points to above) and made sure the same files are changed and usually things are clean without any additional testing.  All the renaming since the 1.11-1.12 bump makes this a bit harder, but it looks like you caught most everything other than (1) the top-level README.rst and (2) GOALS files (though BenA's right that there's no need to modify the CHANGES file which I've already started on).

Then I sometimes check and see if something else was "oops!" updated a few days or weeks after that, but checking the logs, I'm not seeing anything like that this time around, and you caught conf.py which was new since the 1.12 version bump.

I'm noticing some .bad files have 1.12 embedded in them, but I think that the various pre-diff testing scripts paper over differences like this such that it doesn't matter.  Given that a number of other .bads have 1.11 in them, I'm feeling pretty confident in that (nor do I remember anyone having to update .bads in the past).

So with the fix to the .goodstart file, GOALS, and README.rst, I think this would be ready to go.

**Comment by awallace-cray** [comment]
Changes Brad requested have been made.

**Pull request merged**